src/Entity.js

```diff
@@ -0,0 +1,14 @@
+const delim = '$';
+
```

Comment:
why ‘$’, not ‘-‘?

Reply:
`Entity.stringify` is a key. It doesn't know about naming.